python/Scripts/generateshader.py

```diff
@@ -15,7 +15,7 @@ def validateCode(sourceCodeFile, codevalidator, codevalidatorArgs):
     if codevalidator and os.path.isfile(codevalidator):
         cmd = codevalidator + ' ' + sourceCodeFile 
         if codevalidatorArgs:
-            cmd + ' ' + codevalidatorArgs
+            cmd += ' ' + codevalidatorArgs
```

Comment:
This didn't actually add `codevalidatorArgs` to `cmd` before.

Reply:
My bad. Thanks for the fix.